src/claimtrie.cpp

```diff
@@ -137,6 +150,77 @@ void CClaimTrieNode::reorderClaims(supportMapEntryType& supports)
     std::make_heap(claims.begin(), claims.end());
 }
 
+CClaimTrie* CClaimTrie::enableNormalizationFork(bool enable, int nHeight)
+{
+    LogPrintf("%s: Normalize Name Fork is %s\n",
+              __func__, (enable ? "ENABLED" : "DISABLED"));
+
+    if (enable)
```

Comment:
We default this to true after testing but before the release?

Reply:
I don't know what you're asking.  It's a flag specified by a caller.